connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Herder.java

```diff
@@ -144,6 +146,24 @@
      */
     ConnectorStateInfo connectorStatus(String connName);
 
+    /**
+     * Lookup the set of topics currently used by a connector.
+     * @param connName name of the connector
+     */
+    Map<String, ActiveTopicsInfo> connectorActiveTopics(String connName);
```

Comment:
Why return a map keyed by the connector name when `ActiveTopicsInfo` has the connector name in it? I understand that the `ConnectorsResource` is simply calling this, but maybe it'd be better to just return `ActiveTopicsInfo` here -- which seems closer to matching what this method is actually doing, and the fact that `ConnectorsResource` needs a map can be done in that class:
```
    @GET
    @Path("/{connector}/topics")
    public Map<String, ActiveTopicsInfo> getConnectorActiveTopics(final @PathParam("connector") String connector) throws Throwable {
        return herder.connectorActiveTopics(connector);
    }
```
to
```
    @GET
    @Path("/{connector}/topics")
    public Map<String, ActiveTopicsInfo> getConnectorActiveTopics(final @PathParam("connector") String connector) throws Throwable {
        ActiveTopicsInfo result = herder.connectorActiveTopics(connector);
        return Collections.singletonMap(result.connector(), result);
    }
```
This is kind of a minor thing, but since this is an (internal) API it seems better to reflect the actual behavior. WDYT?